setup.py

```diff
@@ -92,13 +94,17 @@ def read(*rnames):
     keywords=keywords,
     packages=[
         "zope",
-        "zope.testing"],
+        "zope.testing",
+    ],
     package_dir={'': 'src'},
     namespace_packages=['zope'],
     install_requires=[
         'setuptools',
-        'zope.exceptions',
-        'zope.interface'],
+    ],
+    extras_require={
+        'test': [
```

Comment:
True, right now it could be considered boilerplate. I like to consider it "standardization" (which is perhaps another name for the same thing 😄 ). All of the recent projects I've worked on have a `test` extra, and many of them have identical (as in, copy-and-pasted) or near identical tox.ini and .travis.yml that enables coveralls and some other things, and which use the `test` extra. I'd ultimately like to get to that point here...but it turns out that certain versions of Python fail the build on a missing `test` extra, but not all of them to. That's probably due to out-of-date setuptools or something (which is something that those configurations update too). So anyway, this seemed like a small step towards that which I discovered here when working with dependencies.